Xamarin.Forms.Core/NavigationPage.cs

```diff
@@ -27,6 +27,8 @@ public class NavigationPage : Page, IPageContainer<Page>, INavigationPageControl
 
 		public static readonly BindableProperty TitleIconProperty = BindableProperty.CreateAttached("TitleIcon", typeof(FileImageSource), typeof(NavigationPage), default(FileImageSource));
 
+		public static readonly BindableProperty TitleViewProperty = BindableProperty.CreateAttached("TitleView", typeof(VisualElement), typeof(NavigationPage), null, propertyChanged: TitleViewPropertyChanged);
```

Comment:
you mean `View` not `VisualElement`, right ? or can I pack a `Page` in there ?

Reply:
...I had a very good reason for changing it from `View` to `VisualElement`, but I cannot for the life of me remember what it was.